naucse/templates/_base.html

```diff
@@ -15,9 +15,6 @@
         <link rel="stylesheet" href="{{url_for('static', filename='css/nausce.css')}}">
         <link rel="stylesheet" href="{{url_for('static', filename='css/body.css')}}">
         <link rel="stylesheet" href="{{url_for('static', filename='css/pygments-lovelace-style.css')}}">
-        {% if var('book-style') %}
-            <link rel="stylesheet" href="{{url_for('static', filename='css/book-style.css')}}">
-        {% endif %}
```

Comment:
Please don't remove the old style for existing courses. People are currently learning from them, the pages shouldn't be changing every week.
The rest of naucse is a playground where we can iterate on the design.